recipes/Data-Prep-Kit/intro/dpk_intro_1_python.ipynb

```diff
@@ -0,0 +1,1123 @@
+        "\n",
+        "These PDF files are created from markdown files using the following command\n",
+        "\n",
+        "```bash\n",
+        "pandoc  earth.md  -o earth.pdf\n",
+        "pandoc  mars.md  -o mars.pdf\n",
+        "```\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "39a0ab6e",
+      "metadata": {
+        "id": "39a0ab6e"
+      },
+      "source": [
+        "## Step-2: Figure out Runtime Environment\n",
+        "\n",
+        "### 2.1 - Determine runtime\n",
+        "\n",
+        "Determine if we are running on Google colab or local python environment"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "1fe354b7",
+      "metadata": {
+        "id": "1fe354b7"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "\n",
+        "if os.getenv(\"COLAB_RELEASE_TAG\"):\n",
+        "   print(\"Running in Colab\")\n",
+        "   RUNNING_IN_COLAB = True\n",
+        "else:\n",
+        "   print(\"NOT in Colab\")\n",
+        "   RUNNING_IN_COLAB = False"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "8e7c104b",
+      "metadata": {
+        "id": "8e7c104b"
+      },
+      "source": [
+        "### 2.2 -Download Data if running on Google Colab"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 30,
+      "id": "3309799e",
+      "metadata": {
+        "id": "3309799e"
+      },
+      "outputs": [],
+      "source": [
+        "if RUNNING_IN_COLAB:\n",
+        "    !mkdir -p 'data/solar-system'\n",
+        "    !wget -O 'data/solar-system/earth.pdf'  'https://raw.githubusercontent.com/sujee/granite-code-cookbook/recipe/data-prep-kit-1-intro/recipes/Data-Prep-Kit/intro/data/solar-system/earth.pdf'\n",
+        "    !wget -O 'data/solar-system/mars.pdf'  'https://raw.githubusercontent.com/sujee/granite-code-cookbook/recipe/data-prep-kit-1-intro/recipes/Data-Prep-Kit/intro/data/solar-system/mars.pdf'\n",
+        "    !wget -O 'utils.py'  'https://raw.githubusercontent.com/sujee/granite-code-cookbook/recipe/data-prep-kit-1-intro/recipes/Data-Prep-Kit/intro/utils.py'\n",
+        "    !wget -O 'requirements.txt'  'https://raw.githubusercontent.com/sujee/granite-code-cookbook/recipe/data-prep-kit-1-intro/recipes/Data-Prep-Kit/intro/requirements.txt'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "a5dc2b68",
+      "metadata": {
+        "id": "a5dc2b68"
+      },
+      "source": [
+        "### 2.3 - Install dependencies if running on Google Colab"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 31,
+      "id": "1fcec577",
+      "metadata": {
+        "id": "1fcec577"
+      },
+      "outputs": [],
+      "source": [
+        "if RUNNING_IN_COLAB:\n",
```

Comment:
The current Jupyter contents in `requirements.txt` shouldn't be installed in Colab. Since @rawkintrevo has worked on our Colab support, ask him what's the best thing to do for installing dependencies.

Reply:
Not to pass the buck, but I don't think I"m doing it right anymore either- have to ask @fayvor